src/components/Features/Transfer/Transfer.js

```diff
@@ -110,6 +117,33 @@ export const Transfer = () => {
     }
   };
 
+  const ReadMore = ({URL, text, openInNewTab = false}) => {
```

Comment:
export this component to the `/UI` directory.
It shouldn't receive `text` since this is `ReadMore` component.

Reply:
I'm not deleting the `text` prop. Maybe we will want that the text be something else than "read more" (e.x "click here for more" or even "Read more")